pkg/resourcemonitor/podresourcesscanner.go

```diff
@@ -61,7 +63,8 @@ func (resMon *PodResourcesScanner) Scan() ([]PodResources, error) {
 	//Pod Resource API client
 	resp, err := resMon.podResourceClient.List(ctx, &podresourcesapi.ListPodResourcesRequest{})
 	if err != nil {
-		return nil, fmt.Errorf("Can't receive response: %v.Get(_) = _, %v", resMon.podResourceClient, err)
+		prometheus.ApiCallsFailure.Inc()
```

Comment:
we need to add labels to distinguish the failures, and we need to track failures of `GetAllocatable` as well (even if we know it will fail only because disabled feature gate, basically).